.github/workflows/update_status.yml

```diff
@@ -84,7 +84,7 @@ jobs:
           conan-center-bot update-status-issue \
             --github-token "$GITHUB_TOKEN" \
             ./status/v1-update.json \
-            https://github.com/qchateau/conan-center-bot/issues/1 \
+            https://github.com/$GITHUB_REPOSITORY/issues/1 \
```

Comment:
it would be better if the issues IDs were set in an environment variable, which would be define in the github project settings

Reply:
Ideally, the user would provide the list of URLs of the issues to update in the environment. I guess a comma separated list would do. It would solve both this and the issue of the official CCI repository.